VenuApp/src/main/java/infinitycodecrew/VenuApp/controllers/WelcomeController.java

```diff
@@ -9,6 +9,6 @@
 public class WelcomeController {
     @GetMapping("/")
     public String welcome(){
-        return "welcome/infinityCodeCrew";
+        return "welcome/welcomePage.html";
```

Comment:
this is changed back in a future commit, I was trying (and failing) to make css work without react and I tried a few alternate paths